packages/create/README.md

```diff
@@ -21,6 +21,15 @@ npx sap-ux
 ## add
 Calling `sap-ux add` allows adding a feature to a project.
 
+## change
+Calling `sap-ux change` allows changing a feature of a project.
+
+### data-source
+Calling `sap-ux change data-source` allows replacing the OData Source of the base application in an adaptation project.  
+```sh
+sap-ux change data-source /path/to/adaptation-project
```

Comment:
Maybe you could mention that the path is optional and that the cwd is used if not provided

Reply:
Mentioned with cd1b20bbdddd1c595d925b5276ce529ca7ab8fa5